modules/nickname/events/guildMemberUpdate.js

```diff
@@ -0,0 +1,53 @@
+const {truncate} = require("../../../src/functions/helpers");
+module.exports.run = async function (client, oldGuildMember, newGuildMember) {
+
+    if (!client.botReadyAt) return;
+    if (newGuildMember.guild.id !== client.guild.id) return;
+    if (newGuildMember.nickname === oldGuildMember.nickname && newGuildMember.roles.highest.position === oldGuildMember.guild.me.roles.highest.position) return;
```

Comment:
`newGuildMember.roles.highest.position === oldGuildMember.guild.me.roles.highest.position`
Could you explain why we're comparing the highest role instead of the hoist role here (I'm not sure myself maybe I'm just stupid, but everywhere else we look at the hoist role)

Reply:
No, I can't. My plan what role to use changed while doing that and I forgot to change. But by switching to highest role now anyways it works, ig